plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java

```diff
@@ -827,6 +884,23 @@ private static boolean isSameMessage(EObject m, SequenceMessage seqMessage) {
     }
     return false;
   }
+  
+
+  private static boolean isSameStateFragment(Scenario scenario, EObject fragment, TimeLapse timelapse) {
```

Comment:
you could remove scenario from parameters, it seems that it's never used

Reply:
Done.